src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
```

Comment:
The import statement for 'java.util.Random' should be placed together with other import statements (if any) without an empty line separating them.